python/ray/services.py

```diff
@@ -1544,7 +1544,7 @@ def start_ray_processes(address_info=None,
             local_scheduler_stdout_file, local_scheduler_stderr_file = (
                 new_log_files(
                     "local_scheduler_{}".format(i),
-                    redirect_output=redirect_worker_output))
+                    redirect_output=redirect_output))
```

Comment:
I think the reason that this is `redirect_worker_output` is that by default the worker output goes to the scheduler output (because the workers are forked from the scheduler), and so if the scheduler output is redirected to a file, then the worker output will be also.

Reply:
yes... because the local scheduler starts the works and `start_worker_command` does not define the redirection of worker log.

However, if I do not redirect the worker log, when I was debugging the local scheduler by log, it would be mixed with the stdout and stderr. And no local_scheduler log files in /tmp/raylogs directory.

So I think the right solution may require to set the worker redirect log files in local scheduler `start_worker_command`.

Anyway, I leave it here because I think it helps me review local scheduler logs. If you need to revert it back, I would do that in the last rebase commit.

Reply:
@chuxi, I think this needs to be reverted, can you revert it?